fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetaClient.java

```diff
@@ -100,6 +101,7 @@ public class HiveMetaClient {
 
     public HiveMetaClient(String uris) throws DdlException {
         HiveConf conf = new HiveConf();
+        conf.addResource(new Path("file:///" + StarRocksFE.STARROCKS_HOME_DIR + "/conf/hive-site.xml"));
```

Comment:
$FE/conf will be used as the classpath of fe. I think it is no necessary to add this?